packages/Operators/test/reference_implementation/DTK_ReferenceHexMesh.cpp

```diff
@@ -50,8 +52,6 @@
 #include <DTK_BasicEntitySet.hpp>
 #include <DTK_EntityCenteredField.hpp>
 
-#include <DTK_DBC.hpp>
```

Comment:
This was moved to `.hpp` file. It probably should have been a part of the commit containing `id()` calls as to make them inline I put them into `hpp` while simultaneously doing some `DTK_REQUIRE` there.